docs/pt/Architecture.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: global
+title: Visão Geral da Arquitetura
+nickname: Visão Geral da Arquitetura
+group: User Guide
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+# Onde o Alluxio Se Encaixa
+
+Devido ao modelo centralizado em memória do Alluxio e ser o ponto central de acesso, Alluxio mantém 
+um lugar único no ecosistema de `Big Data`, residindo entre o armazenamento tradicional como o Amazon S3, 
+Apache HDFS ou OpenStack Swift, e estruturas de computação e aplicações como Apache Spark ou Hadoop 
+MapReduce. Para aplicações de usuários e estruturas de computação, Alluxio é a camada mais baixa que 
+gerencia o acesso ao dado e ao rápido armazenamento destes, facilitando o compartilhamento de dados 
+e a localidade de tarefas, independentemente de onde estas estiverem executando na mesma estrutura 
+computacional. Como resultado, Alluxio pode aumentar em uma ordem de magnitude a velocidade de tarefas 
+para aplicações `Big Data` enquanto provê uma interface comum de acesso ao dado. Já que Alluxio mantém 
+transparente a integração do sistema de armazenamento inferior (`under storage systems`) para as
+aplicações, qualquer armazenamento inferior pode suportar quaisquer aplicações e estruturas sendo 
+executadas em cima do Alluxio. Acompanhado com um potencial que permite montar múltiplas armazenamentos 
+inferiores, Alluxio pode servir como uma camada unificada para as variadas origens de dados. 
+
+![Stack]({{site.data.img.stack}})
+
+# Componentes Alluxio
+
+O modelo do Alluxio utiliza um único `master` e múltiplos `workers`. Em alto nível, Alluxio pode ser 
+divido em três componentes:  o [master](#master); [workers](#worker); e [clients](#client). Os 
+`master` e `workers` juntos formam o Alluxio Server, estes são os componentes que um administrador de
+sistema irá utilizar para gerenciar e manter o sistema. Os `clients` são, genericamente, as aplicações, 
+como as tarefas Spark ou MapReduce, ou usuários utilizando linha de comando (CLI) do Alluxio. Estes 
+usuários irão apenas precisar de uma porção do client do Alluxio.
+
+### Master
+
+Aluxio pode ser disponibilizado em dois modos de `master`, [único master](Running-Alluxio-Locally.html)
```

Comment:
These links don't work currently - lets either add "../en" to link to the English versions, or translate those pages first.